### DISCUSSION TITLE
[Idea] Offer different discounts depending on stock

### DISCUSSION THREAD

**Discussion by Touxten** [PrestaShop/PrestaShop#37566] [Ideas]
### The solution

You would need to add a column to the in the specific table and use the same system as from XX

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
You can use a new hook called `actionValidateCartRule` for this. It allows you to do any validation you can imagine on the cart rule.

**Comment by Touxten** [comment]
Yes but you can't display a price on google adsword with cartrule
